pypeit/core/trace.py

```diff
@@ -988,7 +988,8 @@ def fit_trace(flux, trace_cen, order, ivar=None, bpm=None, trace_bpm=None, weigh
     # NOTE: keck_run_july changes: Added down-weighting of masked parts
     # of the trace.
     # TODO: This feels arbitrary
-    trace_fit_ivar[_trace_bpm] = 0.1
+#    trace_fit_ivar[_trace_bpm] = 0.1
+    trace_fit_ivar[_trace_bpm] = 0.01
```

Comment:
Should this be a Trace parameter?  Seems like it.

Reply:
At the least, make it a method parameter so that you can 
describe what it is and why it is apparently so important.

Reply:
The thing that was causing some NIRES slits to fail was the extrapolation for orders that don't fill the detector. The procedure we adopted which works the best is not to mask the bad pixels entirely in the fits, but rather just replace them with the trace fit from the last iteration and then downweight those pixels. This parameter is changing the weight of those masked pixels to be smaller. This fixes NIRES going haywire, and does not impact other spectrographs. The choice of this parameter was always somewhat arbitrary.

Reply:
Please make the value either a method parameter or
a full PypeItPar parameter.